thunder/torch/__init__.py

```diff
@@ -1875,6 +1875,11 @@ def copysign_(a, b, /):
     return prims.copy_(copysign(a, b), a)
 
 
+@torchsymbol(torch.Tensor.copy_, is_method=True)  # , tags=(prims.OpTags.IN_PLACE,))
```

Comment:
this removal of tags isn't deliberately made. will file a pr to revert this change.

Reply:
okay, this should be like this, meaning w/o in_place tags as in https://dev.azure.com/Lightning-AI/lightning/_build/results?buildId=213793&view=logs&j=3f274fac-2e11-54ca-487e-194c91f3ae9f&t=7d9416f0-1669-5bc4-09c6-d0926f95334e&l=145:
```
+ python -c 'from thunder.executors import nvfuser_available ; assert nvfuser_available(), '\''nvFuser is missing!'\'''
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/__w/2/s/thunder/__init__.py", line 44, in <module>
    from thunder.common import (
  File "/__w/2/s/thunder/common.py", line 40, in <module>
    import thunder.torch as ltorch  # we need to import thunder.torch before the below for import ordering...
  File "/__w/2/s/thunder/torch/__init__.py", line 1879, in <module>
    def copy_(a, b, /):
  File "/__w/2/s/thunder/torch/__init__.py", line 206, in __call__
    _inplace_to_out_of_place[sym] = globals()[name[:-1]], -1
KeyError: 'copy'
```